examples/01-filter/decimate.ipynb

```diff
@@ -0,0 +1,365 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "'\\n.. _decimate_example:\\n\\nDecimation\\n~~~~~~~~~~\\n\\nDecimate a mesh\\n\\n'"
+      ]
+     },
+     "execution_count": 1,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "\"\"\"\n",
+    ".. _decimate_example:\n",
```

Comment:
Just leaving this here so it's harder for us to forget to remove this before finalising the PR.